contracts/InterruptibleInverseVester.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.3;
+import "./InverseVester.sol";
+
+/**
+ * @title Interruptible inverse token vesting contract
+ * @author Inverse Finance
+ * @notice Vesting contract which can be interrupted by governance
+ */
+contract InterruptibleInverseVester is InverseVester {
+    address public governance;
```

Comment:
Can be made `immutable`.

Reply:
might be worth adding a setter tbh

Reply:
I doubt whether it would ever change as the treasury would have to be updated on all smart contracts, but it doesn't hurt to be flexible. Setter would be a good addition!